api/services/errors.go

```diff
@@ -117,6 +117,7 @@ var (
 	ErrAuthInvalid                  = errors.New("auth invalid", ErrLayer, ErrCodeInvalid)
 	ErrAuthUnathorized              = errors.New("auth unauthorized", ErrLayer, ErrCodeUnauthorized)
 	ErrNamespaceLimitReached        = errors.New("namespace limit reached", ErrLayer, ErrCodeLimit)
+	ErrNamespaceZeroReached         = errors.New("namespace creation not permitted for user", ErrLayer, ErrCodeLimit)
```

Comment:
```suggestion
	ErrNamespaceCreationIsForbidden         = errors.New("namespace creation not permitted for user", ErrLayer, ErrCodeLimit)
```

Reply:
In addition to this error, I believe that when it occurs, the route can return a `403` status, allowing the frontend to handle this case appropriately.